projects/ui/src/lib/components/po-field/po-combo/po-combo.component.spec.ts

```diff
@@ -1425,6 +1406,78 @@ describe('PoComboComponent:', () => {
       expect(defaultSpan).toBeTruthy();
     });
 
+    it('should call `onOptionClick` if clicked option isnt`t an option group title', () => {
+      component.options = [{ label: '1', value: '1' }];
+
+      const spyOnOptionClick = spyOn(component, 'onOptionClick');
+
+      fixture.detectChanges();
+
+      const optionItem = component.contentElement.nativeElement.querySelectorAll('li')[0];
+
+      optionItem.click();
+
+      expect(spyOnOptionClick).toHaveBeenCalled();
+    });
+
+    it('shouldn`t call `onOptionClick` if clicked option is an option group title', () => {
+      component.options = [{ label: '1', options: [{ value: 'value'}]}];
+
+      const spyOnOptionClick = spyOn(component, 'onOptionClick');
+
+      fixture.detectChanges();
+
+      const optionItem = component.contentElement.nativeElement.querySelectorAll('li')[0];
+
+      optionItem.click();
+
+      expect(spyOnOptionClick).not.toHaveBeenCalled();
+    });
+
+    it('should contain `po-combo-item` if `comboOptionTemplate` is true and combo options dont`t have groups', () => {
+      component.comboOptionTemplate = <any> { templateRef: null };
+      component.options = [{ label: '1', value: '1' }];
+
+      fixture.detectChanges();
+
+      const comboItemLink = nativeElement.querySelector('.po-combo-item');
+
+      expect(comboItemLink).toBeTruthy();
+    });
+
+    it('shouldn`t contain `po-combo-item` if `comboOptionTemplate` is true but combo options have groups', () => {
+      component.comboOptionTemplate = <any> { templateRef: null };
+      component.options = [{ label: '1', options: [ { value: 'value' } ] }];
+
+      fixture.detectChanges();
+
+      const comboItemLink = nativeElement.querySelector('.po-combo-item');
+
+      expect(comboItemLink).toBeFalsy();
+    });
+
+    it('should contain a tag `po-combo-item-title` if comboOptionTemplate is false and combo options have groups', () => {
```

Comment:
should contain `class`, pois `po-combo-item-title` é uma classe e não uma tag, alterar tbm na linha 1470

Reply:
√